src/trace_processor/importers/proto/statsd_module.cc

```diff
@@ -197,11 +193,12 @@ void StatsdModule::ParseAtom(int64_t ts, protozero::ConstBytes nested_bytes) {
       [&](ArgsTracker::BoundInserter* inserter) {
         ArgsParser delegate(ts, *inserter, *context_->storage);
 
-        const auto& fields = descriptor_->fields();
-        const auto& field_it = fields.find(nested_field_id);
+        const auto& descriptor =
+            context_->descriptor_pool_->descriptors()[*descriptor_idx_];
```

Comment:
are we guaranteed here that `descriptor_idx_` is not `nulllopt`?

Reply:
Made it more clear.